redash/utils/parameterized_query.py

```diff
@@ -96,10 +96,13 @@ def apply(self, parameters):
         return self
 
     def _valid(self, name, value):
+        if not self.schema:
+            return True
```

Comment:
Why is this needed?

Reply:
For the non-textless API?

Reply:
Precisely.

Reply:
Feels like it might bite us in the future, but can't think of any reason why.